Packs/CortexAttackSurfaceManagement/Scripts/InferWhetherServiceIsDev/InferWhetherServiceIsDev.yml

```diff
@@ -0,0 +1,28 @@
+args:
+- description: 'Array of key-value objects. Each object within the array must contain the keys "Key" and "Value" to be considered. The values associated with those keys can be arbitrary. Example: [{"Key": "env", "Value": "dev"}, {"Key": "Name", "Value": "ssh-ec2-machine-name"}]'
+  isArray: true
+  name: asm_tags
+- description: 'Array of strings representing the Xpanse ASM "active classifications" for the service. Example: ["RdpServer", "SelfSignedCertificate"]'
+  isArray: true
+  name: active_classifications
+comment: Identify whether the service is a "development" server. Development servers have no external users and run no production workflows. These servers might be named "dev", but they might also be named "qa", "pre-production", "user acceptance testing", or use other non-production terms. This automation uses both public data visible to anyone (`active_classifications` as derived by Xpanse ASM) as well as checking internal data for AI-learned indicators of development systems (`asm_tags` as derived from integrations with non-public systems).
+commonfields:
+  id: InferWhetherServiceIsDev
+  version: -1
+dockerimage: demisto/python3:3.10.9.45313
+enabled: true
+name: InferWhetherServiceIsDev
+outputs:
+- contextPath: ASM - Dev Check
```

Comment:
the output path `ASM - Dev Check` is wrong
please change to `ASM.IsDevEnvironment`

Reply:
removed ouput: [fixe InferWhetherServiceIsDev and rel notes](https://github.com/demisto/content/pull/23927/commits/83307e4bcc1c992044a437c2ad2f0bb94a2016be)